StardewArchipelago/Locations/CodeInjections/Vanilla/ShippingInjections.cs

```diff
@@ -94,6 +94,7 @@ private void CheckAllShipsanityLocations(List<Item> allShippedItems)
                 {
                     continue;
                 }
+                name = FixArchaeologyLocationInconsistentNaming(name);

```

Comment:
This should happen in the Archaeology section of NameSimplifier, not in Shipsanity code

Reply:
Done.  In fact, the old code stopped working as the internal names changed to be more normal.  They just aren't the same as the english display name.